README.md

```diff
@@ -55,11 +68,16 @@ See the [kibana contributing guide](https://github.com/elastic/kibana/blob/maste
 
 For more information about any of these commands run `yarn ${task} --help`. For a full list of tasks checkout the `package.json` file, or run `yarn run`.
 
-## Installing plugin
+<h2 id="installing-markdown">Installing plugin</h2>
+
+Assuming this plugin is named: "draxis_map_0.0.10.zip"
 
-- log in into your server
+- Log in into your server
 - cd your_kibana_installation_path
-- if you are using docker run: docker exec -it { container_name } bash
-- install plugin by running: bin/kibana-plugin install https://github.com/CUTLER-H2020/Draxis-map-layers-kibana-plugin.git
-- this step takes about 5 minutes
-- done!
+- If you are using docker run:
+
+   `docker exec -it {container_name} bash`
+- Install plugin by running:
+
+  `bin/kibana-plugin install file:///(path/to/)draxis_map_0.0.10.zip`
```

Comment:
Could someone install this plugin directly using https?

eg

https://github.com/CUTLER-H2020/Draxis-map-layers-kibana-plugin/blob/containerization/dist/draxis_map-0.0.10.zip